dandischema/tests/test_utils.py

```diff
@@ -99,6 +100,21 @@ def test_sanitize_value() -> None:
     assert sanitize_value("A\"'B") == "A--B"
 
 
+def test_sort_versions() -> None:
```

Comment:
The use `@pytest.mark.parametrize` can help reduce duplication in logic and make the logic more apparent.